internal/keymagic/publicKeyStore_test.go

```diff
@@ -3,11 +3,12 @@ package keymagic
 import (
 	"context"
 	"fmt"
+	"github.com/quan-to/chevron/internal/agent"
 	"github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/database"
 	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/pkg/models"
 	"github.com/quan-to/chevron/test"
+	"github.com/quan-to/slog"
 	"io/ioutil"
 	"testing"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).